jobs/bump-lockfile.Jenkinsfile

```
@@ -298,6 +302,18 @@ lock(resource: "bump-lockfile") {
             }]}
         }
 
+        stage("Bump date stamp") {
+            // Bump the date stamp file with today date.
+            // This will trigger a konflux build.
+            def trigger_path = ".tekton/trigger"
+            def trigger_comment_path = ".tekton/trigger-file-comment"
+            shwrap("""
+                pushd src/config
+                echo \$(date --utc +%Y%m%d) | cat ${trigger_comment_path} - > ${trigger_path}
+                popd
```

In case it doesn't exist (for whatever reason):

```suggestion
                pushd src/config
                touch ${trigger_comment_path}
                echo \$(date --utc +%Y%m%d) | cat ${trigger_comment_path} - > ${trigger_path}
                popd
```